repo-agent/examples/k8s-repowatch.yaml

```diff
@@ -11,10 +11,11 @@ spec:
       apiKeySecretRef: gemini-vscode-tokens
       prompt: |
         You are a kubernetes core developer who is helping with review comments on a kubernetes repo PR.
-      configdirRef: k8s-gemini-config
+      #configdirRef: k8s-gemini-config
```

Comment:
nit: should we explain what this field does with a comment now that it is commented out? (similar to pullRequests below)